triton/model_repo/greedy_search/config.pbtxt

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: "greedy_search"
+backend: "python"
+max_batch_size: 512
+
+parameters [
+  {
+    key: "context_size",
+    value: { string_value: "2"}
+  },
+  {
+    key: "bpe_model",
+    value: { string_value: "/ws/yuekaiz/icefall-asr-librispeech-pruned-transducer-stateless3-2022-04-29/data/lang_bpe_500/bpe.model"}
```

Comment:
Is this path available in the docker container?

Reply:
Sorry, would update it.